tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -384,80 +209,29 @@ private void includeCss(HttpServletRequest request, PrintWriter out) {
         // Per https://en.wikipedia.org/wiki/JQuery_UI#Release_history --
-        // jquery-ui 1.10.4: Jan 17, 2014
         // jquery-ui 1.12.1: Sep 14, 2016 -- that's the newest
         // Per https://jqueryui.com/ -- Current stable "v1.12.1 jQuery 1.7+"
-        if (doUseDojo) {
-            out.write("<script src='//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js'></script>\n");
-        } else {
-            out.write("<script src='//ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js'></script>\n");
-        }
+        out.write("<script src='//ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js'></script>\n");
     }
 
     private static void includeCldrJavaScript(HttpServletRequest request, Writer out) throws IOException {
         final String prefix = "<script src='" + request.getContextPath() + "/js/";
         final String tail = "'></script>\n";
-        final Boolean doUseDojo = useDojo(request);
-        out.write(String.format("<script>const %s=%s;</script>\n", USE_DOJO_VAR, doUseDojo.toString()));
-
-        if (doUseDojo) {
-            // Autosize 1.18.6 - 2014-03-13
-            out.write(prefix + "jquery.autosize.min.js" + tail);
-        } else {
-            // Autosize 4.0.2 (2018-04-30 per changelog.md), see http://www.jacklmoore.com/autosize
-            out.write(prefix + "autosize.min.js" + tail);
-        }
-
-        if (doUseDojo) {
-            final String js = getCacheBustingExtension(request) + ".js" + tail;
-            out.write(prefix + "CldrDojoText" + js); // CldrDojoText.js
-            out.write(prefix + "CldrDojoStatus" + js); // CldrDojoStatus.js
-            out.write(prefix + "CldrDojoAjax" + js); // CldrDojoAjax.js
-            out.write(prefix + "CldrDojoBulkClosePosts" + js); // CldrDojoBulkClosePosts.js
-            out.write(prefix + "CldrDojoForumParticipation" + js); // CldrDojoForumParticipation.js
-            out.write(prefix + "CldrDojoForumFilter" + js); // CldrDojoForumFilter.js
-            out.write(prefix + "CldrDojoCsvFromTable" + js); // CldrDojoCsvFromTable.js
-            out.write(prefix + "CldrDojoDeferHelp" + js); // CldrDojoDeferHelp.js
-            out.write(prefix + "CldrDojoForum" + js); // CldrDojoForum.js
-            out.write(prefix + "survey" + js); // survey.js
-            out.write(prefix + "CldrDojoLoad" + js); // CldrDojoLoad.js
-            out.write(prefix + "CldrDojoTable" + js); // CldrDojoTable.js
-            out.write(prefix + "bootstrap.min.js" + tail); // exceptional
-            out.write(prefix + "redesign" + js); // redesign.js
-            out.write(prefix + "review" + js); // review.js
-            out.write(prefix + "CldrDojoGui" + js); // CldrGuiDojo.js
-        } else {
-            out.write(prefix + "bootstrap.min.js" + tail);
-        }
+        // Autosize 4.0.2 (2018-04-30 per changelog.md), see http://www.jacklmoore.com/autosize
```

Comment:
same. these should be loaded in the bundle.

Reply:
yes